spec/support/data_generator.rb

```diff
@@ -38,6 +38,7 @@ def generic_user_data
     private_post_as_receipient
     tag_following
     generic_person_data
+    remote_mutual_friend
```

Comment:
Thanks, no need to depend on `conversations` anymore to create a mutual contact ❤️